biz.aQute.bnd/bnd.bnd

```diff
@@ -38,6 +38,7 @@ Bundle-Description: This command line utility is the Swiss army knife of OSGi. I
 -dependson: biz.aQute.bnd.embedded-repo
 
 -buildpath: \
+    org.apache.felix.framework;version=latest,\
```

Comment:
Why is felix framework on buildpath? That  seems odd and potentially dangerous. I don't see anything in this PR that would require this.